lib/routes/mangadex/user/feed.ts

```diff
@@ -0,0 +1,131 @@
+    example: '/mangadex/user/feed/follow/zh?limit=10',
+    radar: [
+        {
+            source: ['mangadex.org/titles/feed'],
+            target: '/user/feed/follow',
+        },
+    ],
+    categories: ['anime'],
+    parameters: {
+        lang: {
+            description: 'The language of the followed manga',
+        },
+    },
+    features: {
+        requireConfig: [
+            {
+                name: 'MANGADEX_USERNAME',
+                description: 'MangaDex Username, required when refresh-token is not set',
+                optional: true,
+            },
+            {
+                name: 'MANGADEX_PASSWORD',
+                description: 'MangaDex Password, required when refresh-token is not set',
+                optional: true,
+            },
+            {
+                name: 'MANGADEX_CLIENT_ID',
+                description: 'MangaDex Client ID',
+                optional: false,
+            },
+            {
+                name: 'MANGADEX_CLIENT_SECRET',
+                description: 'MangaDex Client Secret',
+                optional: false,
+            },
+            {
+                name: 'MANGADEX_REFRESH_TOKEN',
+                description: 'MangaDex Refresh Token, required when username and password are not set',
+                optional: true,
+            },
+            {
+                name: 'MANGADEX_FILTERED_LANGUAGES',
+                description: 'MangaDex Filtered Languages',
+                optional: true,
+            },
+        ],
+    },
+    handler,
+};
+
+async function handler(ctx) {
+    const userFollowUrl = `${constants.API.BASE}/user/follows/manga/feed`;
+
+    const { lang } = ctx.req.param();
+
+    const limit = ctx.req.query('limit') ? Number.parseInt(ctx.req.query('limit')) : DEFAULT_LIMIT;
+
+    const accessToken = await getToken();
+
+    const languagesQuery = new Set([...(typeof lang === 'string' ? [lang] : lang || []), ...(await getFilteredLanguages())].filter(Boolean));
+
+    const feed = (await cache.tryGet(
+        'mangadex:user-follows',
+        async () => {
+            const response = await got.get(
+                `${userFollowUrl}${toQueryString({
+                    translatedLanguage: languagesQuery,
+                    order: {
+                        publishAt: 'desc',
+                    },
+                    limit,
+                })}`,
+                {
+                    headers: {
+                        Authorization: `Bearer ${accessToken}`,
+                        'User-Agent': config.trueUA,
+                    },
+                }
+            );
+
+            const followedChapterFeed = response?.data?.data;
+            if (!followedChapterFeed) {
+                throw new Error('Failed to retrieve user follows from MangaDex API.');
+            }
+
+            return followedChapterFeed;
+        },
+        config.cache.contentExpire,
```

Comment:
https://github.com/DIYgod/RSSHub/pull/18269#discussion_r2072440605
```suggestion
        config.cache.routeExpire,
```
This belongs to a hot cache which requires a shorter TTL.